src/helm/proxy/clients/ai21_client.py

```diff
@@ -20,6 +22,28 @@ class AI21RequestError(Exception):
     pass
 
 
+@dataclass(frozen=True)
```

Comment:
I'm a bit confused why we need the custom model config stuff for AI21 given that is something we define and support as opposed to configured by users?  Does that mean we need another file that we need to pass into HELM to run AI21 evals?

Reply:
Cleaned this up; now we just pass in the additional parameters via `AI21Client.__init__()`